bench/README.md

```diff
@@ -18,40 +28,228 @@ pip3 install -r bench/requirements.txt
 
 ## Preparation
 
+* Copy accounts keys to folder `~/.dcl/keyring-test`:
+* Enter to `bench` folder
+    ```bash
+    cd bench
+    ```
+
+* Open the .env file:
+    * update fields.<br>
+        e.g.:
+        ```yml
+        DCLD_VERSION=v0.11.0
+        DCLD_NODE=tcp://host.docker.internal:26657
+        DCLD_CHAIN_ID=dclchain
+
+        WRITE_HOSTS=http://host.docker.internal:26657
+        READ_HOSTS=http://host.docker.internal:26640
+        TRUSTEE_ACCOUNT_NAME=jack
+        COUNT_USERS=4 
+        ```
```

Comment:
Add clarification for each parameter below pls

Reply:
Done! ✅